.github/workflows/deploy_docs_template.yml

```diff
@@ -3,9 +3,22 @@ name: Deploy docs
 on:
   workflow_call:
 
+env:
+  DOCS_AUTOCOMMIT_TITLE: Docs_examples_update
+  
 jobs:
+  docs_autocommit_check:
```

Comment:
Please name the job, see https://github.com/manticoresoftware/manticoresearch/blob/master/.github/workflows/test.yml as an example